pupil-spa/src/app/services/feedback/feedback.service.ts

```diff
@@ -22,8 +21,8 @@ export class FeedbackService {
   }
 
   async postFeedback() {
-    const storedFeedback = this.storageService.getItem(new FeedbackStorageKey());
-    const accessToken = this.storageService.getItem(new AccessTokenStorageKey());
+    const storedFeedback = this.storageService.getFeedback();
+    const accessToken = this.storageService.getAccessToken();
```

Comment:
getAccessToken() seems quite generic, as we have different tokens for each queue...

Reply:
looking at existing code, its a token being used within the payload of the pupilFeedback queue

Reply:
resolving as per conversation

Reply:
its now removed